docs/development.rst

```diff
@@ -78,6 +79,35 @@ Having the order based on a numeric priority in a common directory allows for mo
 
 These items are subject to change. When making changes to the internal functionality, this page needs to be updated as well.
 
+Working with vendored libs
+--------------------------
+
+Vendored libs are external libraries, meaning source code not maintained by Bash-it
+developers.
+They are ``git subtrees`` curated in the ``vendor/`` folder. To ease the work with git
+vendored libs as subtrees we use the `git-vendor <https://github.com/Tyrben/git-vendor>`_ tool.
```

Comment:
you should add a reference on how to use `git-vendor`, probably can refer to https://github.com/brettlangdon/git-vendor#usage

Reply:
you are referring here to `Tyrben` which is just a lone fork, better reference `brettlangdon`